content/en/docs/reference/config/telemetry/telemetry_v2_with_wasm/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Telemetry V2 with Wasm runtime (Experimental)
+description: How to enable Telemetry V2 with Wasm runtime (experimental).
+weight: 60
+---
+
+In Istio 1.5, by default Telemetry V2 is enabled as compiled in Istio proxy filters. The same filters are also compiled to WebAssembly (Wasm) modules and shipped with Istio proxy. To enable Telemetry V2 with Wasm runtime, apply the following configuration, which is a drop-in replacement for the default Telemetry V2 installation:
+
+{{< text bash >}}
+$ kubectl apply -f https://raw.githubusercontent.com/istio/proxy/release-1.5/extensions/stats/testdata/telemetry-v2-wasm.yaml
```

Comment:
do we need to delete the existing one's that get installed by default?

Reply:
No they are named the same, so this will just config the resource, not create them.